scripts/ci-conformance.sh

```diff
@@ -23,19 +23,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# Install kubectl
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-cd "${REPO_ROOT}" || exit 1
+KUBECTL="hack/tools/bin/kubectl"
```

Comment:
I realize we are cd'ing to repo_root right after, but if a CD happens else where in the script (maybe added in the future) the path could be wrong.  If we prepend this with repo_root we could avoid that type of mistake in future.